Dockerfile

```diff
@@ -41,7 +38,7 @@ COPY --from=go-build /build/trento /app/trento
 LABEL org.opencontainers.image.source="https://github.com/trento-project/trento"
 ENTRYPOINT ["/tini", "--", "/app/trento"]
 
-FROM gcr.io/distroless/base:debug AS trento-web
+FROM registry.suse.com/bci/micro:15.3 AS trento-web
```

Comment:
Did you mean here `registry.suse.com/bci/bci-micro:15.3`?

Reply:
It was still `bci/micro` when I created this PR :see_no_evil: